### DISCUSSION TITLE
usage of Ethernet tethering

### DISCUSSION THREAD

**Discussion by HKL2580** [Mygod/VPNHotspot#548] [Q&A]
I have a type-c hub designed for PC, and my Android version is 11.
  I tried plugging the USB hub into my phone, the type-a port functions perfectly as an OTG port, and the RJ45 will be powered and showed on pc as a functioning card without an IP address or gateway assigned, but neither could I utilize the network from the router nor enable the "Ethernet tethering" function in the app.
  Is this a function only for a certain type of phone or it's my Android system lack functions or the hub's Ethernet card is incompatible (no driver?) with my phone?

**Comment by shivam07maurya** [comment]
Problem:
When routing SIM (cellular) data through Ethernet Tethering from an Android phone to a PC, GTA Online and FiveM fail to connect to game servers — connection to server times out / fails to establish, even though general internet browsing works fine.
Working configurations (no issue):
Phone connected to a second phone's WiFi hotspot → Ethernet Tethering to PC → GTA/FiveM works
SIM data via USB Tethering → GTA/FiveM works
SIM data via phone's own WiFi Hotspot (to PC's WiFi) → GTA/FiveM works
Failing configuration:
SIM (cellular) data → Ethernet Tethering → GTA/FiveM fails to connect to server (browsing/TCP traffic works fine, only game UDP traffic seems affected)
